src/docs/cookbook/navigation/navigation-basics.md

```diff
@@ -19,24 +25,21 @@ So how do we navigate to new screens? Using the [`Navigator`](https://docs.flutt
 
 ## 1. Create two screens
 
-First, we'll need two screens to work with. Since this is a basic example, we'll
-create two screens, each containing a single button. Tapping the button on the
-first screen will navigate to the second screen. Tapping the button on the
-second screen will return our user back to the first!
-
-First, we'll set up the visual structure.
+In this basic example, there are two screens, each containing a single button. 
+- Tapping the button on the first screen will navigate to the second screen. 
```

Comment:
=>  * Tapping the button on the first route navigates to the second route.

Rule: Use present tense:  https://developers.google.com/style/tense